lib/engines/hckinstall/hckinstall.rb

```diff
@@ -304,11 +325,7 @@ def tag
       "install-#{@project.options.install.platform}"
     end
 
-    def run
-      @logger.debug('HCKInstall: run')
-
-      prepare_setup_scripts_config
-
+    def run_studio_installation
```

Comment:
This name is somewhat confusing. `install_studio` is the one which calls `run_studio` so it can be even said that this name is more appropriate for that method.

I think it's good time to rename the existing `install_studio`, `install_client` and `install_clients` to more descriptive names. In particular, the new names should express that those methods consume ISOs created by e.g. `prepare_studio_iso`. For example, `install_studio` can be renamed to `run_studio_installer` (not using the word "iso" here because it's a bit awkward to say "run ISO") and `prepare_studio_iso` can be renamed to `prepare_studio_installer` accordingly.

After this refactoring, `run_studio_installation` can be named as `install_studio`.